mongodb/tests/main_test.bal

```diff
@@ -26,25 +26,27 @@ ConnectionConfig mongoConfig = {
     host: testHostName,
     username: testUser,
     password: testPass,
-    options: {sslEnabled: false, serverSelectionTimeout: 15000}
+    options: {sslEnabled: false, serverSelectionTimeout: 15000},
+    databaseName: COLLECTION_NAME
 };
 
 ConnectionConfig mongoConfigError = {
     options: {
         url: "asdakjdk"
-    }
+    },
+    databaseName: "MyDb"
 };
 
 const DATABASE_NAME = "moviecollection";
 const COLLECTION_NAME = "moviedetails";
```

Comment:
Shall we move these to constants above the `mongoConfig` record initialization (above line no.25)?